app/controllers/idv/review_controller.rb

```diff
@@ -87,6 +87,7 @@ def password
     end
 
     def personal_key_confirmed
+      return unless current_user
```

Comment:
the other filters handle ensuring the user is signed in, etc.

Reply:
Why check this here instead of adding `before_action :confirm_two_factor_authenticated`?